app/services/transaction_search/searcher.rb

```diff
@@ -2,8 +2,22 @@ module TransactionSearch
 
   class Searcher
 
+    # Do not modify this directly. Use TransactionSearch.register instead.
```

Comment:
Why?

Reply:
Adding a new searcher also needs to add an `attr_accessor` to `SearchForm`. The `register` method does that for you.

Reply:
Mind adding that to the comment?